config/schema/rules.action.schema.yml

```diff
@@ -1,3 +1,13 @@
+# For undefined context values just pretend this is a list of strings.
+# We need this to make
+# Drupal\Tests\rules\Kernel\CoreIntegrationTest::testComponentActionAutoSave
+# work.
+rules.action.context_values.*:
+  type: sequence
+  label: Context values
+  sequence:
+    type: string
```

Comment:
Wouldn't it be better to use the type "ignore" then and make it officially optional to have something better?